libclamav/ole2_extract.c

```diff
@@ -1685,15 +1665,11 @@ static cl_error_t handler_otf(ole2_header_t *hdr, property_t *prop, const char *
     ret = ret == CL_VIRUS ? CL_VIRUS : CL_SUCCESS;
 
 done:
-    if (NULL != name) {
-        free(name);
-    }
+    FREE(name);
     if (-1 != ofd) {
         close(ofd);
```

Comment:
Makes you wonder if we should add a `CLOSE_FD()` and a `CLOSE_HANDLE()` (for windows).  Maybe some day, but not today.

Reply:
I was thinking about that as well.  I could add them for this file pretty easily as part of this PR.  Was thinking it could be a good cleanup to start putting these things in as cleanup when I am already changing a file.